Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/LMGs/GSh.yml

```
@@ -0,0 +1,138 @@
+- type: entity
+  parent: RMCBaseWeaponLMG
+  id: RMCGSh_rotary_machinegun
```

Entity IDs should not have underscores in them
```suggestion
  id: RMCWeaponMachinegunGsh
```